flask_appbuilder/security/forms.py

```diff
@@ -60,7 +60,10 @@ class ResetPasswordForm(DynamicForm):
     conf_password = PasswordField(
         lazy_gettext("Confirm Password"),
         description=lazy_gettext("Please rewrite the password to confirm"),
-        validators=[EqualTo("password", message=lazy_gettext("Passwords must match"))],
+        validators=[
+            DataRequired(),
+            EqualTo("password", message=lazy_gettext("Passwords must match")),
+        ],
```

Comment:
if `conf_password` and `password` must be equal, and `password` is required, doesn't imply that `conf_password` is required also?

Reply:
explanation from WTForms ([EqualTo](https://wtforms.readthedocs.io/en/2.3.x/validators/#wtforms.validators.EqualTo))

Reply:
@dpgaspar For you reference from [UserDBModelView](https://github.com/gugupy/Flask-AppBuilder/blob/fab9013003a41c4e80da04f072201a8c7cc99187/flask_appbuilder/security/views.py#L325)